pyfr/readers/base.py

```diff
@@ -22,3 +26,209 @@ def to_pyfrm(self):
         mesh['mesh_uuid'] = np.array(str(uuid.uuid4()), dtype='S')
 
         return mesh
+
+
+class BaseMeshConverter(object):
```

Comment:
I think petype_fnmap is the node numbering of each mesh format and both are the same counter-clockwise first-order node numbering. I move it to each MeshReader.